inc/deploypackage.class.php

```diff
@@ -1646,14 +1646,14 @@ function getPackageForMe($users_id, $computers_id = false) {
       $pfDeployGroup = new PluginFusioninventoryDeployGroup();
       $my_packages   = []; //Store all installable packages
 
-      $query = "";
+      $query = [];
       if ($users_id) {
-         $query = "`users_id`='".$users_id."' AND ";
+         $query += ['users_id' => $users_id];
```

Comment:
Just `$query['users_id'] = $users_id;` would be correct, no? (same for the following +=)